assets/queries/terraform/gcp/stackdriver_logging_disabled/test/positive.tf

```diff
@@ -16,6 +16,18 @@ resource "google_container_cluster" "positive2" {
   initial_node_count = 3
```

Comment:
Since the default behavior is no longer a vulnerability please remove the `positive1` resource

Reply:
ah! also good catch.  ta!